src/Models.hs

```diff
@@ -21,11 +21,12 @@ import           Database.Persist.TH  (mkMigrate, mkPersist, persistLowerCase,
 import           GHC.Generics         (Generic)
 
 import           Config
+import qualified Data.Text as T
```

Comment:
I generally prefer to do

```haskell
import Data.Text (Text)
import qualified Data.Text as Text
```

to make these things more explicit, and also so I don't have to qualify the type name.

Though I *also* prefer to have a centralized `Import` module that does this for you :)